### ISSUE TITLE
Update RPM spec file version to latest (1.2.8)

### ISSUE THREAD

**Issue by pdurbin** [jeffreyhorner/rapache#42]
Hi! Thanks for providing an rApache RPM! I appreciate your [nice note to the community](https://groups.google.com/d/msg/rapache/l7mK-ZjhtfY/QavRlXoD5XIJ).

I noticed that [rapache 1.2.8](https://github.com/jeffreyhorner/rapache/releases/tag/v1.2.8) has been released but the RPM spec file is still at 1.2.7: https://github.com/jeffreyhorner/rapache/blob/v1.2.8/rpm/rapache.spec#L2

I was just building a new rApache RPM for CentOS 7 over at https://github.com/IQSS/dataverse/issues/3208 and would have used rapache 1.2.8 if the spec file had been updated. Instead I built version 1.2.7.

Can the RPM spec file version please be increased to 1.2.8? I figure we should probably use the latest and greatest. Thanks!

**Comment by jeffreyhorner** [comment]
Hi pdurbin,

Would you mind creating a pull request to fix the isue? I'm not a good authority on building RPM packages.

Are you interested in helping maintain rapache, at least for the RPM side?

**Comment by pdurbin** [comment]
@jeffreyhorner are you interested in a pull request that simply changes "1.2.7" to "1.2.8" at https://github.com/jeffreyhorner/rapache/blob/v1.2.8/rpm/rapache.spec#L2 or are you interested in the pull request also containing a little script to build the RPM and a configuration file for an environment in which to build the RPM, such as a Vagrantfile or Dockerfile?

If you take a look at https://github.com/IQSS/dataverse/commit/4daf136b59a5dc7bfbf0f5395194057544128484 you'll see how I used Vagrant to spin up a CentOS VM with which to create an rApache RPM.

I'm suggesting having these files in the repo:

- rpm/rapache.spec (existing)
- rpm/Vagrantfile (new)
- rpm/build.sh (new, I'm not picky about what this file is called)

Basically, you'd type `vagrant up` and it would spin up a CentOS VM, install the rpm build tools, and build the rApache RPM.

**Comment by jeffreyhorner** [comment]
Yes Please! All of it!

**Comment by pdurbin** [comment]
@jeffreyhorner heh. I appreciate the enthusiasm! I'd probably go with Vagrant over Docker just out of familiarity.

@donsizemore and I are discussing this issue, most recently at http://irclog.iq.harvard.edu/dataverse/2016-08-02#i_39007 , and I think we're trying to decide who will make the pull request. We're both pretty busy. 😄 

For the immediate needs of the Dataverse community, I believe the 1.2.7 RPM I built as part of https://github.com/IQSS/dataverse/pull/3210 is sufficient, but I suppose it would be nice to have the latest and greatest (1.2.8).

@jeffreyhorner maybe you can tell me if there are significant changes in 1.2.7 or 1.2.8. (Again, we run 1.2.6 in production.) I don't see either mentioned in https://github.com/jeffreyhorner/rapache/blob/master/ChangeLog but maybe there's a better place to look?

**Comment by jeffreyhorner** [comment]
Literally the only things that have changed are installation and packaging files. The last code fix was in 2013. I'm not sure that's a good or bad thing :)

**Comment by donsizemore** [comment]
My SRPM (all I did to the spec was bump 1.2.7=>1.2.8) is at
http://www.ibiblio.org/dls/rapache-1.2.8-rpm0.src.rpm
used to build
http://www.ibiblio.org/dls/rapache-1.2.8-rpm0.x86_64.rpm
on CentOS-7.2/Apache-2.4.6/R-3.3.1

I tested the lucky numbers locally but admittedly haven't tried it with TwoRavens just yet (my one Cent/RHEL 7 Dataverse box here is our new production one...)

**Comment by donsizemore** [comment]
I should've named the RPMs for arch, sorry. Make that
http://www.ibiblio.org/dls/rapache-1.2.8-el7.src.rpm
and
http://www.ibiblio.org/dls/rapache-1.2.8-el7.x86_64.rpm

**Comment by pdurbin** [comment]
@jeffreyhorner just a heads up that since https://github.com/IQSS/dataverse/pull/3210 has been merged I probably won't work on this issue any time soon. If @donsizemore wants to lead the charge, that's fine. (That pull request does include a working Vagrant environment and a script for building the RPM.) Thanks for letting us know that the actual code hasn't changed since 2013. Thanks for providing a spec file!
